index.html

```diff
@@ -6303,7 +6303,7 @@ <h5>Note regarding the ARIA 1.1 <rref>none</rref> role.</h5>
 				<p>Example use cases:</p>
 				<ul>
 					<li>An element whose content is completely presentational (like a spacer image, decorative graphic, or clearing element);</li>
-					<li>An image that is in a container with the <rref>img</rref> <a>role</a> and where the full text alternative is available and is marked up with <pref>aria-labelledby</pref> and (if needed) <pref>aria-describedby</pref>;</li>
+					<li>An image that is in a container with the <rref>img</rref> <a>role</a> (all children of an <rref>img</rref> are automatically presentational);</li>
```

Comment:
ok... so the wording is definitely an improvement.  but this is still a confusing example.

As I understand this section, this is a list of instances where an author should know to use `role=presentation|none`.  But, if an image is a child of an element with an image role, then it shouldn't be necessary to provide the child image a role of none, since that should happen anyway (really - many implementations treat the children of an image role as if it was completely hidden).  Saying that this is an instance of where one would use role=none|presentation on the child image is just a redundancy.

So, unless i've completely missed the point of this section of content, I think this entire bullet should just be deleted.  Or, maybe if we don't want to lose the bullet, its content could instead be revised to align with this PR https://github.com/w3c/aria/pull/1830